library/system/test/fs_snapshot_test.rb

```diff
@@ -365,4 +401,59 @@ def logger
       end
     end
   end
+
+  describe ".create_snapshot?" do
+    before do
+      Yast.import "Linuxrc"
+    end
+
+    context "when single value is defined on Linuxrc commandline" do
+      it "returns whether given snapshot is requested" do
```

Comment:
I'd change 'snapshot' for 'snapshot type' and maybe 'requested' by 'allowed' (because at this point we only want to know which types are 'allowed').